src/astro/sql/operators/cleanup.py

```diff
@@ -62,7 +62,11 @@ def __init__(
         task_id = task_id or get_unique_task_id("_cleanup")
 
         super().__init__(
-            task_id=task_id, retries=retries, retry_delay=retry_delay, **kwargs
+            task_id=task_id,
+            retries=retries,
+            retry_delay=retry_delay,
+            trigger_rule="all_done",
```

Comment:
@kaxil @dimberman @tatiana 
1. Can we have two tasks with the 'all_done' trigger rule? 
2. If the answer to the previous question is yes, how is the order of execution decided?

Reply:
@utkarsharma2, this looks great. Could you confirm what the DAG looks like in the Airflow interface? If it has the same behaviour as now (that cleanup is not connected to all previous tasks, it feels we should go ahead with this implementation).